webapp/src/pages/FoodCheckout.jsx

```diff
@@ -361,7 +362,7 @@ function OrderSummary({ checkout, chosenInstrument }) {
   // and if there's an error we'll deal with it.
   const showSubmit = checkout.checkoutProhibitedReason !== "charging_prohibited";
   return (
-    <Col xs={12}>
+    <LayoutContainer gutters className="my-4">
```

Comment:
Why are all of these switched to be layout containers? Usually you only want something like that at the root; this is going to be a pain to rearrange, I think.

Reply:
I contemplated with this too, I'll revert this layout change